sdk/monitor/monitor-query/src/internal/modelConverters.ts

```diff
@@ -373,6 +385,35 @@ export function convertGeneratedTable(table: GeneratedTable): LogsTable {
       }
 
       return row;
-    })
+    }),
+    columnDescriptors: table.columns
   };
 }
+
+/**
+ * @internal
+ */
+export function convertBatchQueryResponseHelper(response: GeneratedBatchQueryResponse): any {
+  try {
+    const parsedResponseBody: GeneratedBatchQueryResults = JSON.parse(
+      response.body as any
+    ) as GeneratedBatchQueryResults;
+    return {
+      visualization: parsedResponseBody.render,
+      status: "Success",
```

Comment:
How do you know this is `Success` and not `Partial` or `Failure`?

Reply:
No I am just assigning a value for now, and computing it later. This is just the helper function - this is not the final value returned to user